src/scry/concurrent_rpc.cr

```diff
@@ -0,0 +1,69 @@
+module Scry
+  class ConcurrentRpc
+    CONTEXT = Context.new
```

Comment:
Are you sure it's safe to have the context global to all the fibers?

(not sure what the Context object does in scry)

Reply:
Well,I tried to use `@context` as well, but still doesn't work, see comment below :point_down: